packages/terra-i18n/scripts/post-install/write-default-loaders.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const path = require('path');
+
+if (!fs.existsSync('./node_modules')) {
```

Comment:
This is shady. We should document this behaviour and mention that it will be removed in the next major version bump. (React 17?). It would be weird to me to look in the node modules folder and see some random js files floating around.